css/properties/font.json

```diff
@@ -0,0 +1,149 @@
+{
+  "css": {
+    "properties": {
+      "font": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/font",
+          "support": {
+            "webview_android": {
+              "version_added": null
```

Comment:
true, that's a basic functionality that is supported by every browser (except very old IEs)